### PR TITLE
[Tabs] Show Angular icon properly in tabs

### LINKED ISSUE #6543: Tab icon only show a grey box
**Steps to reproduce**

1. Go to http://todomvc.com/examples/angularjs/#/
1. Open the debugger
1. Open app.js

**Expected results**

I see a "JS" icon before the file name in the tab

**Actual results**

On Nightly 62.0a1 (2018-06-25), I see a grey icon before the file name.

### PR DESCRIPTION
**Pull request by darkwing**
Fixes Issue: #6543

angularfix

### CODE REVIEW

File: src/components/shared/SourceIcon.css [hunk 1]
```diff
@@ -19,6 +19,11 @@
   fill: var(--theme-textbox-box-shadow);
 }
 
+.source-icon.angular {
+  background: url(/images/angular.svg) 1px 1px no-repeat;
+  background-size: 13px 13px;
+}
+
```
Comment (jasonLaster) [inline src/components/shared/SourceIcon.css]: lets also add Vue

https://github.com/devtools-html/debugger.html/blob/master/src/workers/parser/frameworks.js#L17-L19

we might need a good way to safeguard this from future frameworks down the road...

**Comment by darkwing** [comment]
vuefix

**Comment by jasonLaster** [comment]
looks great.

**Comment by jasonLaster** [comment]
By the way, i think if you cleaned up those screenshots so they're more square like, would make for some good twitter material.

**Pull request merged**